multimedia/ffmpeg-devel/Portfile

```diff
@@ -111,12 +108,6 @@ minimum_xcodeversions {9 3.1}
 # requires a C11 compiler
 compiler.c_standard 2011
 
-if {[lsearch [get_canonical_archs] i386] != -1} {
-    # clang-3.1 hits https://trac.macports.org/ticket/30137 (<rdar://problem/11542429>)
-    # clang-139 hits https://trac.macports.org/ticket/38141
-    compiler.blacklist-append {clang < 422.1.7}
-}
-
```

Comment:
I'm okay with this removal, given the later blacklisting of `clang < 800`. Seems like this is outdated cruft that can be removed, unless I'm missing something.